src/transformers/generation/tf_utils.py

```diff
@@ -528,11 +528,11 @@ def compute_transition_scores(
         >>> for tok, score in zip(generated_tokens[0], transition_scores[0]):
         ...     # | token | token string | logits | probability
         ...     print(f"| {tok:5d} | {tokenizer.decode(tok):8s} | {score.numpy():.3f} | {np.exp(score.numpy()):.2%}")
-        |   262 |  the     | -1.413 | 24.33%
-        |  1110 |  day     | -2.609 | 7.36%
-        |   618 |  when    | -2.009 | 13.41%
-        |   356 |  we      | -1.859 | 15.58%
-        |   460 |  can     | -2.508 | 8.14%
+        |   262 |  the     | -1.415 | 24.28%
```

Comment:
small numerical precision errors, so I just rewrote those numbers as they are returned

Reply:
hmm, that's weird. The CI tests are showing slightly different numerical scores, should I just copy the ones returned by the CI?

Reply:
Yes, I'd say stick with the CI values providing they're similar to these. If they're very different, some investigation might be needed, but there's so many small numerical differences that can creep in because of the differences in hardware, setup etc it's probably not worth chasing. We've done the same for some model integration tests